src/main/java/ru/skypro/flea/entity/User.java

```diff
@@ -0,0 +1,48 @@
+package ru.skypro.flea.entity;
+
+import com.sun.istack.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
```

Comment:
pay attention to imports)
it's not the import that supposed to be there
but i've already wrote my thoughts of validation in entities
_i also want to ask your opinion_